README.md

```diff
@@ -15,11 +15,13 @@ type RateLimiter interface {
 }
 ```
 
-The package also provides performance benchmarking; number of operations per number of concurrent goroutines. The following are results of `ratelimit.Benchmark` for different implementations with config aimed at `limit=100, period=10ms` executed on MacBook Pro Quad Core 16GB RAM: 
+The package also provides performance benchmarking; number of operations per number of concurrent goroutines. The following are results of `ratelimit.Benchmark` (in ns/op) for different implementations with config aimed at `limit=100, period=10ms` executed on MacBook Pro Quad Core 16GB RAM: 
```

Comment:
ok i didnt run it on a bad-ass 16GB RAM (🐏 ) computer. So you might wanna re-run this on yours and replace the results 👍